test/protocol-buffer-conversion-test.ts

```diff
@@ -15,7 +37,82 @@ describe('Protocol Buffer Conversion', function(): void {
     const currency = XRPCurrency.from(currencyProto)
 
     // THEN the currency converted as expected.
-    assert.deepEqual(currency.code, currencyProto.getCode())
-    assert.deepEqual(currency.name, currencyProto.getName())
+    assert.deepEqual(currency?.code, currencyProto.getCode())
+    assert.deepEqual(currency?.name, currencyProto.getName())
+  })
+
+  it('Convert PathElement protobuf with all fields set to XRPPathElement', function(): void {
+    // GIVEN a PathElement protocol buffer with all fields set.
+    const pathElementProto = testPathElement
+
+    // WHEN the protocol buffer is converted to a native TypeScript type.
+    const pathElement = XRPPathElement.from(pathElementProto)
+
+    // THEN the currency converted as expected.
+    assert.equal(
+      pathElement?.account,
+      pathElementProto.getAccount()!.getAddress(),
+    )
+    assert.deepEqual(
+      pathElement?.currency,
+      XRPCurrency.from(pathElementProto.getCurrency()!),
+    )
+    assert.equal(
+      pathElement?.issuer,
+      pathElementProto.getIssuer()!.getAddress(),
+    )
+  })
+
+  it('Convert PathElement protobuf with no fields set to XRPPathElement', function(): void {
+    // GIVEN a PathElement protocol buffer with no fields set.
+    const pathElementProto = new Payment.PathElement()
+
+    // WHEN the protocol buffer is converted to a native TypeScript type.
+    const pathElement = XRPPathElement.from(pathElementProto)
+
+    // THEN the currency converted as expected.
+    assert.isUndefined(pathElement?.account)
+    assert.isUndefined(pathElement?.currency)
+    assert.isUndefined(pathElement?.issuer)
+  })
+
+  // MARK: - Org_Xrpl_Rpc_V1_Transaction.Path
```

Comment:
nit: JS doesn't recognize `MARK: -` AFAIK. 

Maybe one of the more knowledgeable JS devs can weigh in if there's a better option.

Reply:
Ah yes, this is deleted in later PRs where I just label each test section with the pb conversion being tested, such as // Path.